src/sweetalert2.scss

```diff
@@ -758,56 +764,47 @@ body {
   }
 }
 
-.swal2-progresssteps {
-  $lightblue: #add8e6;
-  $blue: #3085d6;
-
+.swal2-progress-steps {
   align-items: center;
   margin: $swal2-progress-steps-margin;
-  padding: 0;
-  font-weight: 600;
+  padding: $swal2-progress-steps-padding;
+  background: $swal2-progress-steps-background;
+  font-weight: $swal2-progress-steps-font-weight;
 
   li {
     display: inline-block;
     position: relative;
   }
 
-  .swal2-progresscircle {
-    width: 2em;
-    height: 2em;
-    border-radius: 2em;
-    background: $blue;
-    color: $swal2-white;
-    line-height: 2em;
+  .swal2-progress-step {
+    width: $swal2-progress-steps-step-width;
+    height: $swal2-progress-steps-step-height;
+    border-radius: $swal2-progress-steps-step-border-radius;
+    background: $swal2-progress-steps-active-step-background;
+    color: $swal2-progress-steps-active-step-color;
+    line-height: $swal2-progress-steps-step-height;
     text-align: center;
     z-index: 20;
 
-    &:first-child {
-      margin-left: 0;
-    }
-
-    &:last-child {
-      margin-right: 0;
-    }
-
-    &.swal2-activeprogressstep {
-      background: $blue;
+    &.swal2-active-progress-step {
+      background: $swal2-progress-steps-active-step-background;
 
```

Comment:
In this one the repetition of the `steps` ad `step` in the name, makes more sense, because the property is for `active-step`. So I would leave this like that.